readthedocs/templates/support/index.html

```diff
@@ -0,0 +1,105 @@
+{% extends "base.html" %}
+
+{% load i18n %}
+{% load core_tags %}
+
+
+{% block title %}Support{% endblock %}
+
+{% block content %}
+<div id="support_page">
+
+<h2>Usage Questions</h2>
+
+<p>
+If you have questions about how to use Read the Docs, or have an issue that isn't related to a bug, <a href="http://stackoverflow.com/questions/tagged/read-the-docs">Stack Overflow</a> is the best place to ask. Tag questions with <span class="pre">read-the-docs</span> so other folks can find them easily.
+</p>
+
+<p>
+You might also find the answers you are looking for in our <a href="https://docs.readthedocs.io/page/guides/">documentation guides</a>.
+These provide step-by-step solutions to common user requirements.
+</p>
+
+<h2>Bug Reports</h2>
+
+<p>
+If you have an issue with the actual functioning of the site,
+you can file bug reports on our <a href="https://github.com/readthedocs/readthedocs.org/issues">GitHub issue tracker</a>.
+You can also <a href="http://docs.readthedocs.io/page/contribute.html">contribute</a> to Read the Docs,
+as the code is open source.
+</p>
+
+<h2>Commercial Support</h2>
+
+<p>
+We offer commercial support for Read the Docs
+and commercial hosting with <a href="https://readthedocs.com">Read the Docs for Business</a>,
+We have a dedicated team that responds to support requests during business hours.
+</p>
+
+<h2>User Support</h2>
+
+<p>
+If you need a specific request for your project or account,
+like more resources, change of the project's slug or username.
+Fill out the form below,
+and we will get back to you as soon as possible.
+</p>
+
+<form
+  method="POST"
+  name="fa-form-1"
+  action="https://webhook.frontapp.com/forms/036c4169294f3b04abaa/yuarKDj5Z5LtxfjgyX2G9hA1pZwV8qPBIe1MFXS-nS8TGNSEBTHXHr0D2nsvNb1JltXQKscj55VbOs1Adbli6c-zFS5-rlkrLZZttjBX68XMZX2sACCUmkiBtI0"
```

Comment:
This could be a setting.

Reply:
definitely this should be a setting, we don't want to receive support requests from custom installations p:

Reply:
Aye, good point :)